src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php

```diff
@@ -36,16 +37,21 @@ final class DashboardController
     /** @var RouterInterface */
     private $router;
 
+    /** @var SalesDataProviderInterface|null */
+    private $salesDataProvider;
+
     public function __construct(
         DashboardStatisticsProviderInterface $statisticsProvider,
         ChannelRepositoryInterface $channelRepository,
         EngineInterface $templatingEngine,
-        RouterInterface $router
+        RouterInterface $router,
+        ?SalesDataProviderInterface $salesDataProvider = null
     ) {
         $this->statisticsProvider = $statisticsProvider;
         $this->channelRepository = $channelRepository;
         $this->templatingEngine = $templatingEngine;
         $this->router = $router;
+        $this->salesDataProvider = $salesDataProvider;
```

Comment:
Can we trigger an error if the value is not passed?

Reply:
Added 👍